src/wrapper.ts

```diff
@@ -55,10 +60,10 @@ registerInstrumentations({
   ],
 });
 
-requireIfAvailable([
-  ...MODULES_TO_INSTRUMENT,
-  ...JSON.parse(process.env.MODULES_TO_INSTRUMENT || '[]'),
-]);
+requireIfAvailable(
+  [...MODULES_TO_INSTRUMENT, ...JSON.parse(process.env.MODULES_TO_INSTRUMENT || '[]')],
```

Comment:
This is not acceptable as API naming, and env var are API. I searched for an equivalent OpenTelemetry JS env var and there is none, so let's call it `LUMIGO_JS_DISABLED_INSTRUMENTATIONS` and make it accept a comma-separated list of instrumentation packages that are skipped. (That is, opt-out.) Also, ofc, docs in README.md :-)